CHANGELOG.md

```diff
@@ -29,6 +29,10 @@ binding.navigationView.customizeViewBinders {
     }
 }
 ```
+- :warning: Improved alternatives route processing. `MapboxNavigation#setNavigationRoutes` doesn't accept alternatives which don't deviate from primary route or which have different destinations. Rejected alternatives will be present in `RoutesSetSuccess.ignoredAlternatives` and won't produce `AlternativeRouteMetadata`. [#6464](https://github.com/mapbox/mapbox-navigation-android/pull/6464)
+- Fixed false detection of passed alternatives that led to their removal. [#6464](https://github.com/mapbox/mapbox-navigation-android/pull/6464)
+- Improved route matching. [#6464](https://github.com/mapbox/mapbox-navigation-android/pull/6464)
+- Updated `LocationMatcherResult#zLevel`, `LocationMatcherResult#road` to have no values during `RouteProgressState#OFF_ROUTE` state. [#6464](https://github.com/mapbox/mapbox-navigation-android/pull/6464)
```

Comment:
We should probably mention that the known `MapView` crash, mentioned in 2.9.0-beta.1 release notes, was fixed.

Reply:
Was it fixed? I can't find it in the maps changelog 🤔

Reply:
The error was on Common SDK side and it was fixed in mapbox/mapbox-sdk-common#3116.

Reply:
Thank you
https://github.com/mapbox/mapbox-navigation-android/pull/6486/files